src/client/components/line-chart-axis/line-chart-axis.tsx

```diff
@@ -36,37 +36,42 @@ export interface LineChartAxisProps {
   timezone: Timezone;
 }
 
-export function timeFormat(scale: AxisScale): Unary<Date, string> {
+type TimeFormatter = Unary<moment.Moment, string>;
+
+const formatterFromDef = (format: string): TimeFormatter => time => time.format(format);
+
+export function pickTimeFormatter(scale: AxisScale): TimeFormatter {
   const ticks = scale.ticks();
-  if (ticks.length < 2) return d3.time.format("%c");
+  if (ticks.length < 2) return formatterFromDef("YYYY-MM-DD HH:mm");
   const first = ticks[0];
   const last = ticks[ticks.length - 1];
   if (first.getFullYear() !== last.getFullYear()) {
```

Comment:
This PR is about formatting but we should revisit this function some time in the future.

Right now when picking a formatter for a time span of 2 days with a year boundary included the line chart is presented with 9 ticks every 6 and each of them displays date only.
I will create an issue for this.